stat/distuv/norm.go

```diff
@@ -48,6 +48,12 @@ func (n Normal) CDF(x float64) float64 {
 func (n *Normal) ConjugateUpdate(suffStat []float64, nSamples float64, priorStrength []float64) {
 	// TODO: Support prior strength with math.Inf(1) to allow updating with
 	// a known mean/standard deviation
+	if len(suffStat) != 2 {
```

Comment:
`if len(suffStat) != n.NumSuffStat() {` and update the documentation to that effect.